tests/BenchmarkDotNet.Tests/Exporters/VerifiedFiles/CommonExporterVerifyTests.Exporters_ru-RU.verified.txt

```diff
@@ -429,9 +429,9 @@ Job=LongRun  IterationCount=100  LaunchCount=3
 WarmupCount=15  
 
 {noformat}
-||Method ||    Mean ||Error ||  StdDev ||     P67 ||
-|    Foo | 1,000 ns |    NA | 0,000 ns | 1,000 ns |
-|    Bar | 1,000 ns |    NA | 0,000 ns | 1,000 ns |
+| Method ||Mean     ||Error ||StdDev   ||P67      ||
```

Comment:
Looks like a `|` got removed on accident. I think that's necessary for this format.

Reply:
> Looks like a `|` got removed on accident. I think that's necessary for this format.

Yep, you are right! You can check it again now, it is fixed.